### PR TITLE
halide 16.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/halide.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Halide < Formula
   desc "Language for fast, portable data-parallel computation"
   homepage "https://halide-lang.org"
   # TODO: Check if we can use unversioned `llvm` at version bump.
```
Comment (p-linnane) [inline Formula/halide.rb:4]: Did we try this?

Comment (nandahkrishna) [reply Formula/halide.rb]: Verified locally (Monterey x86_64) that it builds and tests successfully with unversioned `llvm`.

**Comment by chenrui333** [comment]
thanks @nandahkrishna!

**Review by p-linnane — APPROVED** [review]

File: Formula/halide.rb [hunk 2]
```diff
@@ -13,20 +12,20 @@ class Halide < Formula
...
+    sha256 cellar: :any,                 big_sur:        "727ee7167d0f2026de4f7ce5654a601e71485c56573cc766593a748b71849b70"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:   "8ff3882c4b82e3a44a92670c8579b6817447ae304efb6d27c176684e04e9200b"
   end
 
   depends_on "cmake" => :build
   depends_on "pybind11" => :build
   depends_on "jpeg-turbo"
   depends_on "libpng"
-  depends_on "llvm@15"
+  depends_on "llvm"
```
Comment (carlocab) [inline Formula/halide.rb:28]: 👏

**Pull request merged**